src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolJudicialRoleMappingSscsTest.java

```diff
@@ -0,0 +1,416 @@
+package uk.gov.hmcts.reform.orgrolemapping.domain.service;
+
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.kie.api.runtime.rule.QueryResults;
+import org.kie.api.runtime.rule.QueryResultsRow;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.hmcts.reform.orgrolemapping.domain.model.RoleAssignment;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static uk.gov.hmcts.reform.orgrolemapping.domain.service.RequestMappingService.ROLE_ASSIGNMENTS_RESULTS_KEY;
+
+@RunWith(MockitoJUnitRunner.class)
+class DroolJudicialRoleMappingSscsTest extends DroolBase {
+
+    //sscs_caseworker_sscs_judge_org_role
+    @Test
+    void shouldReturnPresidentRoles() {
```

Comment:
Please use Parametrised Test for similar tests

Reply:
Done.